daml-script/daml/Daml/Script.daml

```diff
@@ -121,6 +125,24 @@ data CommandsF a
 newtype Commands a = Commands (Ap CommandsF a)
   deriving (Functor, Applicative)
 
+data DisclosedContract = DisclosedContract
+  with
+    contractId : AnyContractId
+    argument : AnyTemplate
+    metadata : ContractMetadata
+
+data ContractMetadata = ContractMetadata
+  with
+    createdAt : Time
+    keyHash : Text
```

Comment:
maybe worth documenting that it’s base16 encoded binary here?

Reply:
added a doc string.